cmd/dockerd/daemon.go

```diff
@@ -516,7 +516,7 @@ func loadDaemonCliConfig(opts *daemonOptions) (*config.Config, error) {
 	conf.Debug = opts.Debug
 	conf.Hosts = opts.Hosts
 	conf.LogLevel = opts.LogLevel
-	conf.LogFormat = opts.LogFormat
+	conf.LogFormat = log.OutputFormat(opts.LogFormat)
```

Comment:
Perhaps better to move this to a separate PR, for visibility (the change should not strictly be needed as part of the update now)

Reply:
I moved this commit to https://github.com/moby/moby/pull/46525